OpenRA.Mods.D2k/Widgets/Logic/PurchaseWidgetLogic.cs

```diff
@@ -0,0 +1,82 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.Common.Widgets;
+using OpenRA.Widgets;
+
+namespace OpenRA.Mods.D2k.Widgets.Logic
+{
+	public class PurchseWidgetLogic : ChromeLogic
+	{
+		readonly World world;
```

Comment:
```suggestion
		[TranslationReference("time")]
		const string DeliveryIn = "label-deliver-in-timer";
		
		readonly World world;
```

then add the translation key. It will also need to be added to en.ftl